polynote-kernel/src/main/scala/polynote/kernel/util/KernelContext.scala

```diff
@@ -179,13 +181,15 @@ object KernelContext {
     new AbstractFileClassLoader(outputDir, dependencyClassLoader)
   }
 
-  def apply(
+  def default(
     dependencies: Map[String, List[(String, File)]],
+    statusUpdates: Publish[IO, KernelStatusUpdate],
     extraClassPath: List[File]
-  ): KernelContext = apply(dependencies, defaultBaseSettings, extraClassPath, defaultOutputDir, defaultParentClassLoader)
+  ): KernelContext = apply(dependencies, statusUpdates, defaultBaseSettings, extraClassPath, defaultOutputDir, defaultParentClassLoader)
```

Comment:
Is the `statusUpdates` parameter used anywhere?

Reply:
It doesn't seem to be... not sure why I had to do this actually. I guess because `RST` used to get the status updates? Will try to remove at some point and see what happens.